codewars/6kyu/doha22/compare/bench_test.py

```diff
@@ -0,0 +1,25 @@
+from compare import comp
+from compare import comp2
+
+a = [121, 144, 19, 161, 19, 144, 19, 11]
```

Comment:
Try to avoid the global variables as possible, it consider a bad practice for data immutability.
- you should make sure you haven't manipulated the data after doing your processing on it "immutable".
- Solution: define a constant however, Python doesn't have a constant, so an alternative get the data from a function, get_a(), follow the convention of defining "A" constant as a capital letter.